web_src/js/webcomponents/overflow-menu.js

```diff
@@ -60,21 +60,35 @@ window.customElements.define('overflow-menu', class extends HTMLElement {
       this.tippyContent = div;
     }
 
+    const itemFlexSpace = this.menuItemsEl.querySelector('.item-flex-space');
+
     // move items in tippy back into the menu items for subsequent measurement
     for (const item of this.tippyItems || []) {
-      this.menuItemsEl.append(item);
+      if (item.getAttribute('data-after-flex-space')) {
+        this.menuItemsEl.append(item);
+      } else {
+        itemFlexSpace.insertAdjacentElement('beforebegin', item);
```

Comment:
`itemFlexSpace` can be null at this point for menus that don't have that element.

Reply:
No, it is impossible. If there is `data-after-flex-space` attribute, it should be added by a `itemFlexSpace`.

So it can't be null.

Reply:
Hmm, you are right. Will fix it.

Reply:
My point was you are adding `item-flex-space` to only two instances of the component, but there are at least 4 or 5 usages in total, for example user page tabs. It must not crash on `null` there.

Reply:
I see the current code is likely ok now.